src/Tribe/Views/V2/Url.php

```diff
@@ -285,6 +285,16 @@ public static function from_url_and_params( $url = null, array $params = [] ) {
 			$url = home_url( add_query_arg( [] ) );
 		}
 
+		/*
+		 * Depending on the source of the request the encoded chars might be in uppercase (`%C3`) or lowercase (`%c2j`) format.
+		 * Ensure the format of the encoded chars is consistently lowercase.
+		 */
+		$lowercase_encoded_url = preg_replace_callback( '/%[A-F0-9]{2}/', static function ( array $matches ) {
```

Comment:
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: f7b9896270f224e0b6d00b00c6b37eef</sub>